boot/espressif/hal/CMakeLists.txt

```diff
@@ -92,15 +99,34 @@ set(LDFLAGS
     "-lgcov"
     )
 
-add_library(hal STATIC ${hal_srcs} ${INCLUDE_DIRS})
+if (("${MCUBOOT_TARGET}" STREQUAL "xtensa"))
+    list(APPEND CFLAGS
+        "-mlongcalls"
+        )
+    list(APPEND LDFLAGS
+        "-mlongcalls"
+        )
+endif()
 
 set_source_files_properties(
     ${esp_idf_dir}/components/bootloader_support/src/bootloader_flash.c
     ${esp_idf_dir}/components/esp_hw_support/port/${MCUBOOT_TARGET}/rtc_clk_init.c
+    ${esp_idf_dir}/components/esp_hw_support/port/${MCUBOOT_TARGET}/rtc_init.c
     ${esp_idf_dir}/components/esp_hw_support/port/${MCUBOOT_TARGET}/rtc_clk.c
     ${esp_idf_dir}/components/esp_hw_support/port/${MCUBOOT_TARGET}/rtc_time.c
     PROPERTIES COMPILE_FLAGS
     "-Wno-unused-variable")
+    
```

Comment:
Whitespace also here. Please check for this.

Reply:
Done